### ISSUE TITLE
getAllModifications returns campaigns

### ISSUE THREAD

**Issue by guillaumejacquart** [abtasty/flagship-js-sdk#4]
When calling fsVisitor.getAllModifications(), I would expect to have something like a JSON object of all the modifications key/value pairs like so :
```
{
  btnColor: 'red',
  btnText: 'shop now'
}
```

Instead, I get the campaign infos, which requires heavy compute to get modifications (nested structures):
```
// response.data gives this kind of shape:
{
  visitorId: 'VISITOR_ID',
  campaigns: [
    {
      id: 'CAMPAIGN_ID',
      variationGroupId: 'VARIATION_GROUP_ID',
      variation: {
        id: 'VARIATION_ID',
        modifications: {
          type: 'FLAG',
          value: {
            btnColor: '#fff',
          },
        },
      },
    },
    // {...}
  ]
}
```

**Comment by Emidomenge** [comment]
Actually the goal of `fsVisitor.getAllModifications()` is to return the response from the Flagship API as it gives it to us.

We could improve this function just to returns only most interesting data as you mentioned.

I'm thinking about an optional argument `params={simpleMode: true/false}`

**Comment by Emidomenge** [comment]
Requested feature will be available in the next major release 2.0.0
